tests/system/test_cv_configlet.py

```diff
@@ -0,0 +1,300 @@
+        # note this wouldn't show any hours/minutes/seconds
+        return val['name']
+
+@pytest.fixture(scope="class")
+def CvContainerTools_Manager(request):
+    if provision_cv:
+        logger.info('Provision Cloudvision instance')
+        request.cls.provisioner = CloudvisionProvisioner()
+        request.cls.provisioner.configlets_provision(configlets=SYSTEM_CONFIGLETS_TESTS)
+        logger.info('Provisioning done !')
+    logger.info("Execute fixture to create class elements")
+    requests.packages.urllib3.disable_warnings()
+    request.cls.cvp = cvp_login()
+    ansible_module = AnsibleModuleMock(check_mode=False)
+    request.cls.cv_configlet = CvConfigletTools(cv_connection=request.cls.cvp, ansible_module=ansible_module)
+    logger.info("CV connection instance done at {}.".format(time_log()))
+    yield
+    logging.warning('Killing Cloudvision instance')
+
+
+# ---------------------------------------------------------------------------- #
+#   TESTS Management
+# ---------------------------------------------------------------------------- #
+
+
+@pytest.mark.api
+@pytest.mark.configlet
+@pytest.mark.usefixtures("CvContainerTools_Manager")
+@pytest.mark.skipif(user_token == 'unset_token', reason="Token is not set correctly")
+# Parametrize to build a ConfigletInput from list of configlet in SYSTEM_CONFIGLETS_TESTS. Only those set with is_present_expected to False
+@pytest.mark.parametrize("test_configlet", SYSTEM_CONFIGLETS_TESTS, ids=generate_test_ids)
+@pytest.mark.parametrize("check_mode", [True, False], ids=['check_mode_on', 'check_mode_off'])
+class Test_CvConfiglet_Unit():
+
+    @pytest.mark.create
+    @pytest.mark.delete
+    @pytest.mark.dependency(name='authentication')
+    def test_cv_connection(self, test_configlet, check_mode):
+        requests.packages.urllib3.disable_warnings()
+        logger.info("Class is connected to CV at {}".format(time_log()))
+        assert True
+
+    @pytest.mark.parametrize('non_string', [1, 100, True, False])
+    def test_str_cleanup_no_str(self, test_configlet, check_mode, non_string):
+        logger.info('Test str_cleanup with non-str input')
+        res = self.cv_configlet._str_cleanup_line_ending(content=non_string)
+        assert res is None
+        logger.info('Result is None as expected')
+
+    @pytest.mark.dependency(depends=["authentication"], scope='class')
+    def test_configlet_data_from_cv(self, test_configlet, check_mode):
+        # Set check mode
+        self.cv_configlet._ansible.check_mode = check_mode
+        # Work with API
+        logger.info('Start to get configlet data from CV {}'.format(time_log()))
+        cv_data = self.cv_configlet.get_configlet_data_cv(configlet_name=test_configlet['name'])
+        logger.info('Got CV result')
+        logger.info('Got all configlet data from CV {}'.format(time_log()))
+        logger.debug('CV data for configlet {0} is {1}'.format(test_configlet['name'], to_nice_json(data=cv_data)))
```

Comment:
Since this output is only available with `LOGGING` set to `DEBUG`, I would expect to get a maximum of information for troubleshooting. 

Maybe we could change default logging level to INFO and make it user defined (in addition to pytest CLI verbosity) ?

Reply:
Done. Can be managed by using ENV variable to set logging level in pytest.log (Check PR description for usage)